config/webpack/environment.js

```diff
@@ -1,44 +1,33 @@
 const { environment } = require('@rails/webpacker')
 const path = require('path')
 
-// Add global webpack configs here
-
-environment.loaders.delete('css')
-environment.loaders.delete('moduleCss')
-environment.loaders.delete('sass')
-environment.loaders.delete('moduleSass')
+// TS and TSX transpilation is done with tsc via ts-loader, we don't use babel even though it's the
+// webpacker's default. The only use for babel is to transpile legacy module Stats, which could not
+// be migrated to Typescript due to potential extreme sanity loss.
```

Comment:
could you clarify sanity loss? Sanity of the code or sanity of the coder?

Reply:
> **Sanity.**
> the state of having a healthy mind and not being mentally ill

The coder of course 😆